hyundai_kia_connect_api/Vehicle.py

```diff
@@ -284,6 +292,26 @@ def ev_estimated_station_charge_duration(self, value):
         self._ev_estimated_station_charge_duration_unit = value[1]
         self._ev_estimated_station_charge_duration = value[0]
 
+    @property
+    def ev_target_range_charge_AC(self):
+        return self._ev_target_range_charge_AC
+
+    @ev_target_range_charge_AC.setter
+    def ev_target_range_charge_AC(self, value):
+        self._ev_target_range_charge_AC_value = value[0]
+        self._ev_target_range_charge_AC_unit = value[1]
+        self._ev_target_range_charge_AC = value[0]
```

Comment:
Why use 2 attributes to store the same value?

The 2 attributes in question being _ev_target_range_charge_AC_value and _ev_target_range_charge_AC

Same remark for DC

Reply:
Following the rest of the file.  Beyond that I don't know.